src/room/ReactionAudioRenderer.tsx

```diff
@@ -5,70 +5,55 @@ SPDX-License-Identifier: AGPL-3.0-only
 Please see LICENSE in the repository root for full details.
 */
 
-import { ReactNode, useEffect, useRef } from "react";
+import { ReactNode, useDeferredValue, useEffect } from "react";
 
 import { useReactions } from "../useReactions";
-import {
-  playReactionsSound,
-  soundEffectVolumeSetting as effectSoundVolumeSetting,
-  useSetting,
-} from "../settings/settings";
+import { playReactionsSound, useSetting } from "../settings/settings";
 import { GenericReaction, ReactionSet } from "../reactions";
+import { prefetchSounds, useAudioContext } from "../useAudioContext";
+import { useLatest } from "../useLatest";
+
+const SoundMap = Object.fromEntries([
+  ...ReactionSet.filter((v) => v.sound !== undefined).map((v) => [
+    v.name,
+    v.sound!,
+  ]),
+  [GenericReaction.name, GenericReaction.sound],
+]);
+
+const Sounds = prefetchSounds(SoundMap);
```

Comment:
Should we only be pre-fetching if the user has the play reaction sounds setting turned on? At the moment it loads even if I have it disabled.

Reply:
Hm, actually in the case of reactions it's probably fine to not preload them anyway since unlike call join/leaves you *probably* won't be trying to play them before you have loaded them.

Let's load on first use.